src/NuGet.Core/NuGet.Versioning/FloatRange.cs

```diff
@@ -202,17 +197,16 @@ public static bool TryParse(string versionString, out FloatRange range)
                         {
                             releasePrefix = actualVersion.Substring(versionString.LastIndexOf('-') + 1);
 
-                            if (actualVersion.EndsWith("-"))
+                            // For numeric labels 0 is the lowest. For alpha-numeric - is the lowest.
+                            if (releasePrefix.Length == 0 || actualVersion.EndsWith("."))
```

Comment:
The previous code had a comment around `actualVersion.EndsWith(".") `- `// TODO: Solve this better.`

Have we addressed that? If not then we should leave the comment there.